### PR TITLE
Nicer error message if dependencies are not installed

### PR DESCRIPTION
**Pull request by cebe**
added a note that running composer install is required.

### CODE REVIEW

File: validator.php [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 // validates that all security advisories are valid
 
-require __DIR__.'/vendor/autoload.php';
+if (!is_file($autoloader = __DIR__.'/vendor/autoload.php')) {
```
Comment (Ocramius) [inline validator.php]: This... is kind of a normal thing in a composer project.

Feels like you're fighting the wrong error to me, or completely misunderstood what kind of project we're talking about...

Comment (cebe) [inline validator.php]: yeah but this repo is for any kind of php user who maybe is not aware of composer.

the contrib readme states the following:

> Make sure your file validates by running php validator.php from the root of this project.

It does not mention composer at all. It should be either there in the readme or here. Both would be maximum friendly to newbies.

Comment (stof) [inline validator.php]: Please also add it in the readme

Comment (cebe) [inline validator.php]: done.

**Comment by fabpot** [comment]
Thank you @cebe.

**Pull request merged**